docs/components/Page.js

```diff
@@ -50,18 +53,20 @@ export default function Page({ title: page, children }: Props): Node {
         </Box>
       </Box>
 
-      <Box
-        minWidth={200}
-        maxWidth={240}
-        marginStart={4}
-        mdMarginStart={6}
-        lgMarginStart={8}
-        display="none"
-        lgDisplay="block"
-        flex="none"
-      >
-        <Toc cards={sections} />
-      </Box>
+      {!isFullWidth && (
```

Comment:
@ayeshakmaz isn't it odd that when `isFullWidth` is `true`, we don't render the sections?

Reply:
Maybe isFullWidth isn't the best name, but this is essentially saying if the page content is full width and doesn't have sections, than don't render the right-side nav

Reply:
Perhaps the name should be more direct then, like `hideRightNav`?